values.yaml

```diff
@@ -75,7 +75,12 @@ global:
   defaultDenyNetworkPolicy: true
 
   # Enable security context constraints required for OpenShift
-  sccEnabled: false
+  sccEnabled: true
```

Comment:
default is false here as well i suppose

Reply:
default is false for ocp it will be true

Reply:
updated values.yaml and pushed the changes with default values